src/pages/dashboard/AllLineschart/AllLinesChart.tsx

```diff
@@ -20,15 +20,31 @@ const convertToChartCompatibleStruct = (arr: GroupByRes[]) => {
 interface AllChartComponentProps {
   startDate: Dayjs
   endDate: Dayjs
+  alertAllChartsZeroLinesHandling: (arg: boolean) => void
 }
 
-export const AllLinesChart: FC<AllChartComponentProps> = ({ startDate, endDate }) => {
+export const AllLinesChart: FC<AllChartComponentProps> = ({
+  startDate,
+  endDate,
+  alertAllChartsZeroLinesHandling,
+}) => {
   const [groupByOperatorData, groupByOperatorLoading] = useGroupBy({
     dateTo: endDate,
     dateFrom: startDate,
     groupBy: 'operator_ref',
   })
   const { t } = useTranslation()
+
+  useEffect(() => {
+    const totalElements = groupByOperatorData.length
+    const totalZeroElements = groupByOperatorData.filter((el) => el.total_actual_rides === 0).length
+    if (totalElements === 0 || totalZeroElements === totalElements) {
+      alertAllChartsZeroLinesHandling(true)
+    } else {
+      alertAllChartsZeroLinesHandling(false)
+    }
+  }, [groupByOperatorData])
```

Comment:
https://react.dev/learn/you-might-not-need-an-effect#passing-data-to-the-parent

It feels a little bit odd to pass data to the parent.
I'm not sure - should we move the fetching to the parent?

Anyway, I'd feel better if it will use a context instead of callback